code/modules/food/beverages/bottler.dm

```diff
@@ -0,0 +1,450 @@
+				available_recipes += recipe
+				acceptable_items |= recipe.obj_1_item
+				acceptable_items |= recipe.obj_2_item
+				acceptable_items |= recipe.obj_3_item
+			else
+				qdel(recipe)
+
+/obj/machinery/bottler/attackby(obj/item/O as obj, mob/user as mob, params)
+	if(istype(O, /obj/item/weapon/wrench))		//This being before the NODROP check allows borgs to (un)wrench bottlers in case they need move them to fix stuff
+		playsound(src, 'sound/items/Ratchet.ogg', 50, 1)
+		if(anchored)
+			anchored = 0
+			to_chat(user, "<span class='alert'>\The [src] can now be moved.</span>")
+			return
+		else if(!anchored)
+			anchored = 1
+			to_chat(user, "<span class='alert'>\The [src] is now secured.</span>")
+			return
+	if(O.flags & NODROP)
+		to_chat(user, "<span class='warning'>\The [O] is stuck to your hand, you can't seem to put it down!</span>")
+		return
+	if(is_type_in_list(O,acceptable_items))
+		if(istype(O, /obj/item/weapon/reagent_containers/food/snacks))
+			var/obj/item/weapon/reagent_containers/food/snacks/S = O
+			user.unEquip(S)
+			if(S.reagents && !S.reagents.total_volume)		//This prevents us from using empty foods, should one occur due to some sort of error
+				to_chat(user, "<span class='warning'>\The [S.name] is gone, oh no!</span>")
+				qdel(S)			//Delete the food object because it is useless even as food due to the lack of reagents
+			else
+				insert_item(S, user)
+			return
+		else if(istype(O, /obj/item/weapon/reagent_containers/food/drinks/cans))
+			var/obj/item/weapon/reagent_containers/food/drinks/cans/C = O
+			if(C.reagents)
+				if(C.canopened && C.reagents.total_volume)		//This prevents us from using opened cans that still have something in them
+					to_chat(user, "<span class='warning'>Only unopened cans and bottles can be processed to ensure product integrity.</span>")
+					return
+				user.unEquip(C)
+				if(!C.reagents.total_volume)		//Empty cans get recycled, even if they have somehow remained unopened due to some sort of error
+					recycle_container(C)
+				else								//Full cans that are unopened get inserted for processing as ingredients
+					insert_item(C, user)
+			return
+		else
+			user.unEquip(O)
+			insert_item(O, user)
+			return
+	else if(istype(O, /obj/item/trash/can))			//Crushed cans (and bottles) are returnable still
+		var/obj/item/trash/can/C = O
+		user.unEquip(C)
+		recycle_container(C)
+		return
+	else if(istype(O, /obj/item/stack/sheet))		//Sheets of materials can replenish the machine's supply of drink containers (when people inevitably don't return them)
+		var/obj/item/stack/sheet/S = O
+		user.unEquip(S)
+		process_sheets(S)
+		return
+	else		//If it doesn't qualify in the above checks, we don't want it. Inform the person so they (ideally) stop trying to put the nuke disc in.
+		to_chat(user, "<span class='warning'>You aren't sure this is able to be processed by the machine.</span>")
+		return
+	//..()
+
+/obj/machinery/bottler/proc/insert_item(obj/item/O as obj, mob/user as mob)
```

Comment:
they are not following the existing standard for new code, and they are pointless remenants that are hard to mass replace because they still have usage for verbs